src/rapids_pre_commit_hooks/lint.py

```diff
@@ -16,14 +16,24 @@
 import bisect
 import contextlib
 import functools
-import itertools
 import re
 import warnings
 
 from rich.console import Console
 from rich.markup import escape
 
 
+# Taken from Python docs
+# (https://docs.python.org/3.12/library/itertools.html#itertools.pairwise)
```

Comment:
```suggestion
# (https://docs.python.org/3.12/library/itertools.html#itertools.pairwise)
# Replace with itertools.pairwise after dropping Python 3.9 support
```

Reply:
I've added the comment. Note that when we drop 3.9, support, we will probably just revert this entire PR.